Shoko.Server/Providers/TMDB/TmdbMetadataService.cs

```diff
@@ -2072,11 +2116,11 @@ private async Task DownloadPersonImages(int personId, ProfileImages images, bool
         await _imageService.DownloadImagesByType(images.Profiles, ImageEntityType.Person, ForeignEntityType.Person, personId, settings.TMDB.MaxAutoStaffImages, [], forceDownload);
     }
 
-    public async Task<bool> PurgePerson(int personId, bool removeImageFiles = true)
+    public async Task<bool> PurgePerson(int personId, bool removeImageFiles = true, bool forceRemoval = false)
```

Comment:
Really don't like the forced removal. as said earlier, we should be attempting to fix it more gracefully if possible, if the person we're missing has any cast/crew credits locally, instead of just attempting to purge all crew/cast relations. by updating the linked entities it will naturally remove the person once all references are gone.